src/content/tools/dart-compile.md

```diff
@@ -178,11 +231,17 @@ see the platform documentation for those operating systems:
 [`SignTool.exe` documentation]: https://docs.microsoft.com/dotnet/framework/tools/signtool-exe
 [Apple Code Signing guide]: {{site.apple-dev}}/support/code-signing/
 
-#### Known limitations
+#### Known limitations {: #known-limitations }
```

Comment:
I'd also note that cross-compilation support is limited and link to Cross-compilation header.

Reply:
Added. Take a close look at my changes.